build-app.sh

```diff
@@ -1 +1,3 @@
+#!/bin/sh
```

Comment:
Better to use `#!/usr/bin/env sh` or even `#!/usr/bin/env bash` to use the recommended shell of this environment.